app/webpacker/src/stylesheets/_commodity-codes.scss

```diff
@@ -61,6 +61,11 @@
   }
 }
 
+.govuk-tag--green--wide {
+  @extend .govuk-tag--green;
+  max-width: 100%;
```

Comment:
This is to avoid the tag to stay in just one line (without it, the tag renders in two lines)